app/web_conn.go

```diff
@@ -249,8 +358,8 @@ func (wc *WebConn) writePump() {
 			buf.Reset()
 			var err error
 			if evtOk {
-				cpyEvt := evt.SetSequence(wc.Sequence)
-				err = cpyEvt.Encode(enc)
+				evt = evt.SetSequence(wc.Sequence)
+				err = evt.Encode(enc)
```

Comment:
```suggestion
				err = evt.SetSequence(wc.Sequence).Encode(enc)
```

Reply:
Actually, we shouldn't do this. `evt` is "immutable", and `SetSequence` returns a new event, which is used later in the code. So if we don't re-assign `evt`, then it will use the wrong sequence number.

Reply:
That's absolutely right :+1: